app/services/ItemsCachingService.scala

```diff
@@ -86,31 +86,22 @@ class ItemsCachingService @Inject()(cacheService: CustomsCacheService)(appConfig
       .getEntry[Seq[PackageInformation]](PackageInformation.formId)
       .map(_.zipWithIndex.map {
         case (packageInfo, index) =>
-          createPackaging(packageInfo, index)
+          PackagingBuilder.createPackaging(packageInfo, index)
       })
 
-  private def createPackaging(packageInfo: PackageInformation, index: Int) =
-    models.declaration.governmentagencygoodsitem.Packaging(
-      sequenceNumeric = Some(index),
-      typeCode = packageInfo.typesOfPackages,
-      quantity = packageInfo.numberOfPackages,
-      marksNumbersId = packageInfo.shippingMarks
-    )
-
   def procedureCodes(
     cachedData: CacheMap
   ): Option[Seq[models.declaration.governmentagencygoodsitem.GovernmentProcedure]] =
     cachedData
       .getEntry[ProcedureCodesData](ProcedureCodesData.formId)
```

Comment:
Are we keeping this still?

Reply:
ItemCachingService will cease to exist soon so no point in changing this.